src/library/scala/collection/mutable/package.scala

```diff
@@ -2,10 +2,6 @@ package scala.collection
 
 
 package object mutable {
-  @deprecated("Use WrappedArray instead of ArraySeq; it can represent both, boxed and unboxed arrays", "2.13.0")
-  type ArraySeq[X] = WrappedArray[X]
-  @deprecated("Use WrappedArray instead of ArraySeq; it can represent both, boxed and unboxed arrays", "2.13.0")
-  val ArraySeq = WrappedArray.untagged
```

Comment:
I think we need the deprecated aliases in the other direction, if we're doing this change.

Reply:
Yes, thanks for pointing that out. Though I think we should drop the second commit, keep `mutable.WrappedArray`. Waiting for opinions on that still :-)